www/index.php

```diff
@@ -140,9 +140,9 @@
             <div id="test_box-container">
                 <ul class="ui-tabs-nav">
                     <li class="analytical_review ui-state-default ui-corner-top ui-tabs-selected ui-state-active"><a href="#">Advanced Testing</a></li>
-                    <li class="easy_mode"><a href=\"/easy.php\">Simple Testing</a></li>
-                    <li class="visual_comparison"><a href="/video/">Visual Comparison</a></li>
-                    <li class="traceroute"><a href="/traceroute.php">Traceroute</a></li>
+                    <li class="easy_mode"><a href="easy.php">Simple Testing</a></li>

```

Comment:
WPT doesn't support relative paths.  You need to keep the absolute paths in place.

Reply:
if correct edit to a href ="/easymode" ?

Reply:
if is echo not show text

Reply:
no, it is "/easy.php" or "/easy".  There is no /easymode rewrite rule

Reply:
ok i edit finish thank you

Reply:
Can PR complete the merge.